cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -523,29 +523,67 @@ func generateInitramfsMounts() error {
 	return fmt.Errorf("internal error: mode in generateInitramfsMounts not handled")
 }
 
-func unlockIfEncrypted(name string) (string, error) {
+var (
+	secbootLockAccessToSealedKeys = secboot.LockAccessToSealedKeys
+)
+
+// unlockIfEncrypted verifies if an encrypted volume with the specified name exists and unlocks
+// it if this is the case. If this is the last device to be unlocked the access to the sealed keys
+// will be locked. The path to the unencrypted device node is returned.
+func unlockIfEncrypted(name string, last bool) (string, error) {
```

Comment:
Given the explaination here, maybe instead of "last" (which does not give too much information) something like "lockKeyOnFinish" or something more descriptive what will happen?